lib/glific/clients/avanti.ex

```diff
@@ -21,13 +21,11 @@ defmodule Glific.Clients.Avanti do
     {:ok, now} = "Asia/Kolkata" |> DateTime.now()
```

Comment:
not a great use of pipe either, so do:

{:ok, now} = DateTime.now("Asia/Kolkata")

and something similar for the statement below